docs-md/README.md

```diff
@@ -14,7 +14,7 @@ The SDK leverages Alchemy's hardened node infrastructure, guaranteeing best-in-c
 
 The SDK currently supports the following chains:
 
-- **Ethereum**: Mainnet, Goerli
+- **Ethereum**: Mainnet, Goerli, Sepolia
```

Comment:
Can you revert the changes in `/docs-md`? The docs represent the current released version and are autogenerated before cutting a new version

Reply:
oh sweet, ok will revert